### PR TITLE
Details on previewing the message sent via rabbit MQ

### PR DESCRIPTION
**Pull request by dwhenry**
https://trello.com/c/KDfyOeS1/331-specialist-publisher-implementation

### CODE REVIEW

File: source/manual/rabbitmq.html.md [hunk 1]
```diff
@@ -78,3 +78,14 @@ Note: there is a risk that you might delete the wrong message(s). This
...
+
+## Previewing a message for a document_type
+
+As the message which is published for each document type can vary based on
+what infomation is passed in from the publishing app, this allows us to easily
+generate an example message based off the last published message for a given
+document type:
+
+```
+bundle exec rake queue:preview_last_message[<document_type>]
```
Comment (cbaines) [inline source/manual/rabbitmq.html.md]: Sounds good, but it's unclear where this rake task sits?

**Comment by MatMoore** [comment]
I think this needs a bit more context since the page is about rabbitmq in general. That command is something you run against the publishing api, if you're interested in whats published to the `published_documents` exchange. Don't think that's obvious because we use rabbitmq for some other things as well.

File: source/manual/rabbitmq.html.md [hunk 2]
```diff
@@ -78,3 +78,14 @@ Note: there is a risk that you might delete the wrong message(s). This
 is because the contents of the queue may have changed.
 
 Repeat, but change the "Requeue" option to "No".
+
+## Previewing a message for a document_type
+
+As the message which is published for each document type can vary based on
+what infomation is passed in from the publishing app, this allows us to easily
+generate an example message using a recently published message (based off
+last public_updated_At) for a given document type:
```
Comment (MatMoore) [inline source/manual/rabbitmq.html.md]: `public_updated_at` should be lowercase here.

**Comment by dwhenry** [comment]
@MatMoore updated and added content.

File: source/manual/rabbitmq.html.md [hunk 3]
```diff
@@ -78,3 +78,19 @@ Note: there is a risk that you might delete the wrong message(s). This
 is because the contents of the queue may have changed.
 
 Repeat, but change the "Requeue" option to "No".
+
+## Previewing a message for a document_type
+
+The publishing API generates messages when content is updated and posts them
+to the rabbit MQ exchange. Each message has a shared format, however teh contents
+of the message is affected by the publoishing app and what data it sends to the
+publioshing API.
```
Comment (MatMoore) [inline source/manual/rabbitmq.html.md]: There's a few typos here, can you do another edit?

- publoishing/publishging
- teh
- RabbitMQ
- to allow easily generate
- generetd
- As message for different formats can vary

I'd also explicitly name the exchange (published_documents) so people can easily track down the messages in the management UI.

Comment (dwhenry) [reply source/manual/rabbitmq.html.md]: @brenetic can you pair with me on this so we can get it merged. :)

**Comment by brenetic** [comment]
@MatMoore Updated for your viewing pleasure.

**Comment by Rosa-Fox** [comment]
This is really useful! We have been using it today in integration. Will wait for @MatMoore  to give the final approval though.

**Pull request merged**